frontend/packages/core/src/Charts/helpers.tsx

```diff
@@ -20,7 +26,17 @@ const getMinAndMaxOfRangeUsingKey = (data: any, key: string) => {
 export const calculateDomainEdges = (data: any, dataKey: string, edgeRatio: number) => {
   // Get the max and min of the domain, then calculate a certain amount`out from each edge.
   const { min, max } = getMinAndMaxOfRangeUsingKey(data, dataKey);
-  if (edgeRatio <= 0) {
+  if (
+    edgeRatio <= 0 ||
+    min === null ||
+    max === null ||
+    min === Infinity ||
+    max === -Infinity ||
+    min === undefined ||
+    max === undefined ||
+    Number.isNaN(min) ||
+    Number.isNaN(max)
```

Comment:
i believe this can be simplified using `Number.isFinite`

edit: same for other cases below